tst/EnergyPlus/unit/UnitaryHybridAirConditioner.unit.cc

```diff
@@ -1265,4 +1265,124 @@ TEST_F(EnergyPlusFixture, DISABLED_Test_UnitaryHybridAirConditioner_ModelOperati
     }
 }
 
+TEST_F(EnergyPlusFixture, DISABLED_Test_UnitaryHybridAirConditioner_ValidateError)

```

Comment:
Please enable this unit test and all of the other hybrid unitary tests. They run fine locally here.